packages/@aws-cdk/cfnspec/spec-source/000_CloudFormationResourceSpecification.json

```diff
@@ -47447,6 +47447,12 @@
           "Required": true,
           "UpdateType": "Immutable"
         },
+        "EncryptionKey": {
```

Comment:
Please don't edit this file. This represents the "true" form of the current spec; we use the patch files (see the other files in this dir) to update the spec for us.

Reply:
Reverted changes